DiagManager/Pristine/Multicopy.ps1

```diff
@@ -0,0 +1,45 @@
+﻿param(
+  [Parameter(Position=0)]
+  [string]$sourcePath,
+  [Parameter(Position=1)]
+  [string]$destinationPath,
+  [Parameter(Position=2)]
+  [string]$serverName
+)
+
+function Get-InstanceNameOnly([string]$NetnamePlusInstance)
+{
+    try 
+    {
+        $selectedSqlInstance = $NetnamePlusInstance.Substring($NetnamePlusInstance.IndexOf("\") + 1)
+        return $selectedSqlInstance         
+    }
+    catch 
+    {
+        Write-Output "Exception in Get-InstanceNameOnly: $_.Exception.Message)"
+    }
+}
+
+if ($serverName -Like '*\*')
+{
+  $serverName = Get-InstanceNameOnly -NetnamePlusInstance $serverName 
+}
+
+$errorFile = "c:\temp\copyFileError.txt"
```

Comment:
I would keep the $.Exception.Message part and either send it to the console with Write-Output , or if you want to send to a file, replace the c:\temp with a file in the internal folder %internal_output_name% -is variable for it so better pass it as a parameter, The file can be named something like $serverName + "" + TaskName (best to pass this as a parameter into the script)